src/frontend/Sider/Behandling/RevurderFra/VedtakshistorikkTabellVisning.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { Table } from '@navikt/ds-react';
+
+const VedtakshistorikkTabellVisning: React.FC<{
```

Comment:
Denne blir unik for tilsyn barn og burde plasseres under komponentene til tilsyn barn

Reply:
Jo, men de inneholder ikke den samme dataen. Og har kanskke forskjellige behov. Tilsyn barn har `antallBarn` som de andre ikke har

Reply:
Skjønner! 👍 
Renamer komponenter her: 0b2908db8885efc29ab7c7dca12aa7ba5e0ef324

Reply:
Legger de nye komponentene over i sin egen mappe for TilsynBarn, og legger på en sjekk i RevurderFra så de bare vises dersom behandlingen gjelder TilsynBarn: 89049939e587d9b4ec54ae1846c718bbad34aa3c